core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionFileSystemOutput.java

```diff
@@ -134,8 +134,10 @@ public TransmissionFileSystemOutput(String folderPath) {
 
     @Override
     public boolean send(Transmission transmission) {
-        if (size.get() >= capacityInKB) {
-        	InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.WARN, "Persistent storage max capacity has been reached; currently at %s KB. Telemetry will be lost, please set the MaxTransmissionStorageFilesCapacityInMB property in the configuration file.", size.get());
+
+        long currentSizeInBytes = size.get();
+        if (currentSizeInBytes >= capacityInBytes) {
+        	InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.WARN, "Persistent storage max capacity has been reached; currently at %s KB. Telemetry will be lost, please set the MaxTransmissionStorageFilesCapacityInMB property in the configuration file.", (currentSizeInBytes / 1024));
```

Comment:
Sorry i didn't notice this earlier, change this to `%.3f`; and change `1024` to `1024.0` so the quotient is a double.

Also, nitpicking, the user doesn't need to "`please set`" the property; the should "`consider changing`" it. i.e. hitting this message doesn't necessarily mean something is wrong.

Reply:
Thanks done!